src/SamplesApp/SamplesApp.Wasm/SamplesApp.Wasm.csproj

```diff
@@ -11,6 +11,8 @@
 		
 		<!-- https://github.com/unoplatform/uno/issues/61 -->
 		<UnoSkipUserControlsInVisualTree>false</UnoSkipUserControlsInVisualTree>
+		<MonoWasmSDKUri Condition="!$([MSBuild]::IsOsPlatform('Linux'))">C:\Temp\mono-wasm-80c6ee45de6.zip</MonoWasmSDKUri>
```

Comment:
☝️ Don't think this should have been committed? Local debug hooks.

Reply:
It should not have :)